cmd/controlloop/controlloop.go

```diff
@@ -180,3 +194,16 @@ func cronExpressionFromFlatFile() string {
 	}
 	return flatipam.IPAM.ReconcilerCronExpression
 }
+
+// nodeExists will return true if a node with the provided nodeName is found, false otherwise. The returned error will
+// be nil for NotFound errors, otherwise the error from the Nodes().Get() method will be returned.
+func nodeExists(k8sClientSet *kubernetes.Clientset, nodeName string) (bool, error) {
```

Comment:
I think this signature is confusing.

Can't we just return an error and have the client compare against nil for knowing if the node exists ?

Reply:
done